src/Collection.php

```diff
@@ -209,7 +209,7 @@ public function aggregate(array $pipeline, array $options = [])
         if ( ! isset($options['readConcern']) &&
             ! ($hasOutStage && $this->readConcern->getLevel() === ReadConcern::MAJORITY) &&
             \MongoDB\server_supports_feature($server, self::$wireVersionForReadConcern) &&
-            ! $this->isInTransaction($options)) {
+            ! \MongoDB\is_in_transaction($options)) {
```

Comment:
[PHPLIB-429](https://jira.mongodb.org/browse/PHPLIB-429) dropped support for 5.5, so if you'd like to import the function you can. Alternatively, we can wait to do this across the board in a later PR.

Reply:
I'd like to do this across the board and bring in a slightly softer variant of https://github.com/doctrine/coding-standard that works with PHP 5.